projects/ad469x_fmcz/src/ad469x_iio.h

```diff
@@ -0,0 +1,45 @@
+#ifndef AD469X_IIO
+#define AD469X_IIO
+
+#include "iio_types.h"
+#include "ad469x.h"
+
+
+struct scan_type ad469x_scan_type = {
```

Comment:
`ad469x_scan_type` structure is also declared in c file. Probably we should remove from here